lib/rules/key-spacing.js

```diff
@@ -333,11 +336,18 @@ module.exports.schema = [
             "align": {
                 "enum": ["colon", "value"]
             },
+            "mode": {
+                "enum": ["strict", "minimum"]
+            },
             "beforeColon": {
-                "type": "boolean"
+                "type": ["boolean", "number"],
```

Comment:
I agree.  I'm not convinced about the value of forcing a particular minimum number of spaces.  Also, this wouldn't address the original issue reporter's use case of indenting to the next indentation level (total column position equaling next multiple of 2 or 4 or... well it's based on indent setting of the text editor.).